migrations/20170101000000_initial.js

```diff
@@ -0,0 +1,49 @@
+exports.up = (knex, Promise) => {
+  const schoolsTable = knex.schema.createTable('schools', (table) => {
+    table.increments('id');
+    table.string('name').notNullable().unique();
+    table.string('abbreviation', 32);
+  });
+
+  const departmentsTable = knex.schema.createTable('departments', (table) => {
+    table.uuid('id').notNullable().primary();
```

Comment:
Not sure if this is a good idea. There are some advantage and disadvantage to using UUID for PK, but I'm worried about consistency in terms of different tables having different column types for their PK, which would make it to write new schema and work with the data

Reply:
I was about to go with uuids for every table. Auto increment id then?

Reply:
I think that would work, yes